doc/user/installation.rst

```diff
@@ -382,6 +386,10 @@ options to the configuration script.
    Configure zebra to use `dir` for local state files, such as pid files and
    unix sockets.
 
+.. option:: --with-scriptdir <dir>
+
+   Look for Lua scripts in ``dir`` [``prefix``/etc/frr].
```

Comment:
shouldn't this be /etc/frr/scripts ? 
if not maybe s/<dir>/<scripts_parent_dir>

Reply:
fixed